plugins/memory/memory_osx

```diff
@@ -35,33 +34,68 @@ if [ "$1" = "autoconf" ]; then
     fi
 fi
 
-TOTALMEM=$(sysctl hw.memsize | cut -d" " -f2)
-graphlimit=$TOTALMEM
-
 dehumanise() {
     echo "$1" | sed -e "s/K/*1024/g;s/M/*1024*1024/;s/G/*1024*1024*1024/;s/T/*1024*1024*1024*1024/"
```

Comment:
Just as a recommendation for clearer code (and for avoiding `bc`):
```shell
local value="$1"
case "$value" in
    *K) echo "$((value * 1024))";;
    *M) echo "$((value * 1024 * 1024))";;
    *G) echo "$((value * 1024 * 1024 * 1024))";;
    *T) echo "$((value * 1024 * 1024 * 1024 * 1024))";;
    *) echo "$value";;
esac
```

But feel free to keep it unchanged, if you prefer.

Reply:
unfortunately it won't work exactly that way because the output contains decimal places, floating point values, such as 1.23.